cmd/mr_create.go

```diff
@@ -312,6 +242,23 @@ func runMRCreate(cmd *cobra.Command, args []string) {
 		log.Fatal("empty MR message")
 	}
 
+	closeIssues, err := cmd.Flags().GetStringSlice("close-issue")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	if len(closeIssues) > 0 {
+		closeIssueString := "\n\nCloses "
```

Comment:
What happens if `Closes` is in the first line? Does it still work?
This is the Gitlab regex just for reference `/^\/(?<cmd>close|reopen|...)(?:( |$))(?<arg>[^\/\n]*)(?:\n|$)/`
which if you take all the fuzz, it matches that it is at the start of the line.